Swift/ReplicatorConfiguration.swift

```diff
@@ -422,6 +422,14 @@ public struct ReplicatorConfiguration {
         c.maxAttemptWaitTime = self.maxAttemptWaitTime
         c.enableAutoPurge = self.enableAutoPurge
         
+        for (col, config) in self.collectionConfigs {
+            if !col.isValid {
+                fatalError("Tries to add invalid collection")
```

Comment:
For consistency, use the same error message in objective-c : `Attempt to add an invalid collection`.

Reply:
```suggestion
                fatalError("Attempt to add an invalid collection")
```